assets/src/edit-story/components/elementLink/test/output.js

```diff
@@ -14,81 +14,67 @@
+    const props = {
+      element: {
+        id: '123',
+        type: 'image',
+        mimeType: 'image/png',
+        scale: 1,
+        x: 50,
+        y: 100,
+        height: 1920,
+        width: 1080,
+        rotationAngle: 0,
+        resource: {
           type: 'image',
           mimeType: 'image/png',
-          scale: 1,
-          x: 50,
-          y: 100,
+          src: 'https://example.com/image.png',
           height: 1920,
           width: 1080,
-          rotationAngle: 0,
-          resource: {
-            type: 'image',
-            mimeType: 'image/png',
-            src: 'https://example.com/image.png',
-            height: 1920,
-            width: 1080,
-          },
-          link: {
-            url: 'https://example.com/',
-            icon: 'https://example.com/image.png',
-            desc: 'Lorem ipsum dolor',
-            type: 2,
-          },
         },
-      };
+        link: {
+          url: 'https://example.com/',
+          icon: 'https://example.com/image.png',
+          desc: 'Lorem ipsum dolor',
+          type: 2,
+        },
+      },
+    };
+    return (
+      <WithLink {...props}>
+        <amp-img src="https://example.com/image.png" layout="fill" />
+      </WithLink>
+    );
+  }
 
-      await expect(
-        <WithLink {...props}>
-          <amp-img src="https://example.com/image.png" layout="fill" />
-        </WithLink>
-      ).toBeValidAMPStoryElement();
+  describe('a[target]', () => {
+    it('should use target=_blank', async () => {
+      const { container } = render(withLink());
+      const a = container.querySelector('a');
+      await expect(a.target).toBe('_blank');
+      await expect(a.rel).toBe('noreferrer');
     });
+  });
 
-    it('should produce valid AMP output for one-tap links', async () => {
-      const props = {
-        element: {
-          id: '123',
-          type: 'image',
-          mimeType: 'image/png',
-          scale: 1,
-          x: 50,
-          y: 100,
-          height: 1920,
-          width: 1080,
-          rotationAngle: 0,
-          resource: {
-            type: 'image',
-            mimeType: 'image/png',
-            src: 'https://example.com/image.png',
-            height: 1920,
-            width: 1080,
-          },
-          link: {
-            url: 'https://example.com/',
-            icon: 'https://example.com/image.png',
-            desc: 'Lorem ipsum dolor',
-            type: 1,
-          },
-        },
-      };
+  describe('AMP validation', () => {
+    it('should produce valid AMP output', async () => {
+      await expect(withLink()).toBeValidAMPStoryElement();
+    });
 
-      await expect(
-        <WithLink {...props}>
-          <amp-img src="https://example.com/image.png" layout="fill" />
-        </WithLink>
-      ).toBeValidAMPStoryElement();
+    it('should produce valid AMP output for one-tap links', async () => {
+      await expect(withLink()).toBeValidAMPStoryElement();
     });
```

Comment:
What's the difference between these two test cases?

Reply:
This test (`should produce valid AMP output for one-tap links`) can actually be removed. We don't have one-tap links anymore.

That also means removing `type: 2,` from the `props` above.